### PR TITLE
Question copy

### PR DESCRIPTION
**Pull request by JagadishPujari**
https://project-sunbird.atlassian.net/browse/SB-4911
1. Make a copy of a question
2. User can copy others question and save

### CODE REVIEW

File: editor/questionbankcontroller.js [hunk 1]
```diff
@@ -267,7 +268,7 @@ angular.module('createquestionapp', [])
       }
 
       var filterMetaData = {};
-      ecEditor.dispatchEvent('org.ekstep.editcontentmeta:showpopup', { action: 'question-filter-view', subType: 'questions', framework: ecEditor.getContext('framework'), rootOrgId: ecEditor.getContext('channel'), type: 'content', popup: false, metadata: filterMetaData});
+      ecEditor.dispatchEvent('org.ekstep.editcontentmeta:showpopup', { action: 'question-filter-view', subType: 'questions', framework: "NCF", rootOrgId: ecEditor.getContext('channel'), type: 'content', popup: false, metadata: filterMetaData});
```
Comment (manojchandrashekar) [inline editor/questionbankcontroller.js]: Why have you committed hard-coded values? Please verify your changes before you commit!

File: editor/questionbankcontroller.js [hunk 2]
```diff
@@ -501,7 +502,7 @@ angular.module('createquestionapp', [])
       delete $scope.selectedQueIndex;
       $scope.selectedQueIndex = index;
       var filterMetaData = {};
-      ecEditor.dispatchEvent('org.ekstep.editcontentmeta:showpopup', { action: 'question-filter-view', subType: 'questions', framework: ecEditor.getContext('framework'), rootOrgId: ecEditor.getContext('channel'), type: 'content', popup: false, metadata: filterMetaData});
+      ecEditor.dispatchEvent('org.ekstep.editcontentmeta:showpopup', { action: 'question-filter-view', subType: 'questions', framework: "NCF", rootOrgId: ecEditor.getContext('channel'), type: 'content', popup: false, metadata: filterMetaData});
```
Comment (manojchandrashekar) [inline editor/questionbankcontroller.js]: Why have you committed hard-coded values? Please verify your changes before you commit!

File: package.json [hunk 3]
```diff
@@ -5,5 +5,8 @@
     },
     "devDependencies": {
         "ekstep-content-plugin-dev-common": "ekstep/ekstep-content-plugin-dev-common"
+    },
+    "dependencies": {
+        "gulp": "^3.9.1"
```
Comment (manojchandrashekar) [inline package.json]: Why is this needed?

File: editor/questionbankcontroller.js [hunk 4]
```diff
@@ -520,6 +521,105 @@ angular.module('createquestionapp', [])
         ecEditor.dispatchEvent($scope.pluginIdObj.question_create_id + ":showpopup", questionObj);
       }
     }
+    $scope.copyQuestion = function(questionObj){
```
Comment (manojchandrashekar) [inline editor/questionbankcontroller.js:534]: Why have you repeated the save question code here? Can't we make it a function and call it in both places?

**Review by manojchandrashekar — CHANGES_REQUESTED** [review]

**Review by vinukumar-vs — APPROVED** [review]

**Pull request merged**